src/lib.rs

```diff
@@ -5,13 +5,38 @@ pub use serde_json_core;
 
 pub use derive_miniconf::{Miniconf, MiniconfAtomic};
 
+/// Errors that occur during settings configuration
 #[derive(Debug, PartialEq)]
 pub enum Error {
+    /// The provided name wasn't found in the structure.
```

Comment:
Shouldn't this be `key` instead of `name` for consistency everywhere?
And this should all be present tense.

Reply:
This is using `name` in the docs because the error codes currently use the "name" terminology. I have no preference either way - technically, it refers to a variable name in the struct, but it's also a key to accessing a struct member, so I think both are logically correct.

Reply:
Ok. But then shouldn't the stabilizer docs also use "name" to refer to the miniconf keys?

Reply:
Or "path"?

Reply:
I'm honestly of the opinion here to rename all the errors to refer to it as a path, as it _is_ a path-like object to get to the member.

Reply:
Agreed.
Matches https://github.com/quartiq/stabilizer/blob/master/miniconf.py